framework/material/Navigator.kt

```diff
@@ -420,6 +422,7 @@ object Navigator {
         state: NavigatorState<T>,
         modifier: Modifier = Modifier,
         itemHeight: Dp = ITEM_HEIGHT,
+        horizontalItemPadding: Dp = 0.dp,
```

Comment:
This results in a potentially variable amount of padding, but all we really want to encapsulate is two possibilities: having an icon, or having some padding to make up for the lack of an icon.

Therefore we should instead make the padding exist, or not exist, based on the presence, or lack therefore, of an `iconArg`

Reply:
Done.
```kt
val horizontalItemPadding = if (iconArg == null) 0.dp else Dialog.DIALOG_SPACING
```